lib/crawler_web/controllers/user_auth.ex

```diff
@@ -0,0 +1,94 @@
+defmodule CrawlerWeb.UserAuth do
```

Comment:
```suggestion
defmodule CrawlerWeb.UserAuthController do
```

Reply:
Fixed in [ad04701](https://github.com/vnntsu/elixir-ic/pull/32/commits/ad047011f81fa1690a0b30b06d42c2428b04616d)